lib/server.js

```diff
@@ -48,6 +48,11 @@ function createServer (options, httpHandler) {
     } else {
       listenOptions.cb = cb
     }
+    const onAborted = () => { this.close() }
```

Comment:
I now I am nagging, but you dont need to instantiate onAborted when no .signal was supplied by the listenOptions.signal. So you should move the onAborted into the if block.

Reply:
That's ok, you are absolutely right